src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -245,6 +245,37 @@ public override string ToString()
         /// <returns>The <see cref="ControlState"/> containing the result of the addition.</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static ControlState operator +(ControlState lhs, ControlState rhs) => Create(lhs, rhs);
+
+        /// <summary>
+        /// The substraction operator.
+        /// </summary>
+        /// <param name="lhs">A <see cref="ControlState"/> on the left hand side.</param>
+        /// <param name="rhs">A <see cref="ControlState"/> on the right hand side.</param>
+        /// <returns>The <see cref="ControlState"/> containing the result of the substraction.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ControlState operator -(ControlState lhs, ControlState rhs)
```

Comment:
Hello, could you explain why this - operator overriding is added and what this is for?

Reply:
I think this is useful to remove specific ControlState from combined ControlState. for example,
Pressed State:
```cs
ControlState += ControlState.Pressed;
```
UnPressed State:
```cs
ControlState -= ControlState.Pressed;
```
Also, it is used at line 1125 in ViewInternal.cs

Reply:
Thank you for nice explanation~